docs/detections/prebuilt-rules/rule-details/suspicious-file-creation-in-etc-for-persistence.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[suspicious-file-creation-in-etc-for-persistence]]
+=== Suspicious File Creation in /etc for Persistence
+
+Detects the manual creation of files in specific etc directories, via user root, used by Linux malware to persist and elevate privileges on compromised systems. File creation in these directories should not be entirely common and could indicate a malicious binary or script installing persistence for long term access.
```

Comment:
```suggestion
Detects the manual creation of files in specific etc directories via user root. Is used by Linux malware to persist and elevate privileges on compromised systems. File creation in these directories should not be entirely common and could indicate a malicious binary or script installing persistence for long term access.
```

Reply:
Flagging this because the first setence didn't quite make sense to me. Feel free to revise this further if needed.